pkgs/by-name/ma/mactop/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "mactop";
+  version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "context-labs";
+    repo = "mactop";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-BcBUOI5EE04ZTPoHGrNQjctsDFbMoe/6MZaLj/58c34=";
+  };
+
+  vendorHash = "sha256-/KecVx4Gp776t8gFSO29E1q9v29nwrKIWZYCpj7IlSo=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = {
+    description = "Apple Silicon Monitor Top written in pure Golang";
```

The following might be a more useful description:
```suggestion
    description = "Terminal-based monitoring tool "top" designed to display real-time metrics for Apple Silicon chips";
```